cmd/exo/cmd/vm_reboot.go

```diff
@@ -0,0 +1,80 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/exoscale/egoscale"
+	"github.com/spf13/cobra"
+)
+
+// rebootCmd represents the reboot command
+var vmRebootCmd = &cobra.Command{
+	Use:   "reboot <vm name> [vm name] ...",
+	Short: "Reboot virtual machine instance",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			return cmd.Usage()
+		}
+
+		for _, v := range args {
+			vm, err := getVMWithNameOrID(cs, v)
+			if err != nil {
+				return err
+			}
+
+			fmt.Printf("Rebooting %q ", vm.Name)
+			if err := vm.asyncRebootWithCtx(gContext, *cs, true); err != nil {
+				fmt.Fprintln(os.Stderr, err)
+			} else {
+				fmt.Println("\nRebooted !")
+			}
+		}
+		return nil
```

Comment:
1. you have printf in many places, move it all here
2. it'd group the errors in one place and return them

```golang
errors := make([]error, 0)
for _, ... := range args {
    ...
        fmt.Println("failure!")
        errors = append(errors, err)
    ...
        fmt.Println("success.")
}

if len(errors) == 1 {
    return errors[0]
}
if len(errors) > 1 {
    return fmt.Errorf("many errors ... ", ...)
}
return nil
```